src/components/menu-top/MenuTop.vue

```diff
@@ -136,6 +144,28 @@ export default Vue.extend({
       },
     },
   },
+  methods: {
+    saveFile(): void {
+      const filename = this.$store.getters.getShowTitle + '.shw';
```

Comment:
Keep in mind that show titles may have spaces in them. I think file names should be able to support spaces but double check it.

Reply:
I tested and it looks like spaces in the filename is OK